packages/client/src/LocClient.ts

```diff
@@ -190,17 +191,30 @@ export interface AddLinkParams {
 }
 
 export function withLocId<T>(locId: UUID, params: BlockchainSubmission<T>): BlockchainSubmission<T & FetchParameters> {
+    return addToPayload({ locId }, params)
+}
+
+export function addToPayload<T, P>(additionalPayload: P, params: BlockchainSubmission<T>): BlockchainSubmission<T & P> {
```

Comment:
You could use the same convention was  `withLocId` e.g. `withAdditional`.

Reply:
Done